clients/admin-ui/src/features/system/system-form-declaration-tab/PrivacyDeclarationDisplayGroup.tsx

```diff
@@ -141,18 +145,20 @@ export const PrivacyDeclarationDisplayGroup = ({
         ) : null
       }
       footerButton={
-        <Button
-          onClick={handleAdd}
-          size="xs"
-          px={2}
-          py={1}
-          backgroundColor="primary.800"
-          color="white"
-          fontWeight="600"
-          rightIcon={<AddIcon />}
-        >
-          Add data use
-        </Button>
+        !lockedForGVL ? (
+          <Button
```

Comment:
this does get us into a state where if a user already added gvl systems but no data uses, they won't be able to change that now. but hopefully that's okay

Reply:
Hmm I wonder if the "syncs" @adamsachs is working on would resolve this issue..

Reply:
yup, that would get taken care of by the subsequent GVL/compass --> fides sync. the data uses would get populated based on the GVL definition. currently Dawn and i have that sync set up to run once a day.